package.json

```diff
@@ -10,10 +10,12 @@
     "clean": "rimraf './dist/*!(.gitkeep)' './webpack-assets.json'",
     "dev:admin": "better-npm-run dev:admin",
     "dev:amo": "better-npm-run dev:amo",
+    "dev:amo:proxy": "better-npm-run dev:amo:proxy",
```

Comment:
The last line when this starts up is:
````
INFO: server/35663 on indigo.local: 👁  Open your browser at http://127.0.0.1:3333 to view it. (app=amo)
````
Is there an easy way to output another line mentioning the proxy? Maybe not.